api_default.go

```diff
@@ -0,0 +1,237 @@
+/*
+OCPI versions module
+
+Specification for OCPIs versions handlers
+
+API version: 2.2.1
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
+
+package OCPI
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"net/url"
+)
+
+
+// DefaultApiService DefaultApi service
+type DefaultApiService service
+
+type ApiOcpi221DetailsGetRequest struct {
+	ctx context.Context
+	ApiService *DefaultApiService
```

Comment:
There's only one Service - for versions. Other API's aren't included for some reason

Reply:
fixed ✅